projects/plugins/super-cache/inc/delete-cache-button.php

```diff
@@ -86,6 +86,17 @@ function wpsc_admin_bar_delete_cache_ajax() {
 		if ( defined( 'WPSCDELETEERROR' ) ) {
 			return json_decode( constant( 'WPSCDELETEERROR' ) );
 		} else {
+			$req_path = !empty($_POST['path']) ? $_POST['path'] : '/';
+			$referer = !empty($_SERVER['HTTP_REFERER']) ? $_SERVER['HTTP_REFERER'] : admin_url();
```

Comment:
It's probably safer to use `wp_get_referer()` rather than accessing $_SERVER directly.

Reply:
good point, changed that :)